From 093d56c4894e447609dc93d100a0b61a48f178e3 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 18 Nov 2023 11:03:08 +0100
Subject: [PATCH] build: import "gl" dependency
References: https://github.com/CuarzoSoftware/SRM/pull/8

[    3s] ../lib/SRMBuffer.c:1:10: fatal error: GL/gl.h: No such file or directory

Looking for the GLESv2 component is not enough to ensure gl.h exists.
The Mesa packages considers them to be separate entities (because different
.pc files), and distributions readily split Mesa into individually-installable
libGL and libGLESv2 components, too.
---
 src/meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/meson.build b/src/meson.build
index e3cc862..e710f12 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -57,6 +57,7 @@ endforeach
 # ------------ DEPENDENCIES ------------
 
 pkg              = import('pkgconfig')
+gl_dep           = dependency('gl')
 egl_dep          = dependency('egl')
 glesv2_dep       = dependency('glesv2')
 udev_dep         = dependency('libudev')
@@ -77,6 +78,7 @@ SRM = library(
     dependencies : [
         display_info_dep,
         egl_dep,
+        gl_dep,
         glesv2_dep,
         udev_dep,
         pthread_dep,
-- 
2.42.1

